services/web/server/tests/unit/with_dbs/04/products/test_products_service.py

```diff
@@ -194,3 +196,26 @@ async def test_auto_create_products_groups(app: web.Application):
     assert all(
         group_id is not None for group_id in groups.values()
     ), f"Invalid {groups}"
+
+
+async def test_get_product_api_base_url(
```

Comment:
No, not like this. You should have a test where you call your function from `packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/webserver/products.py`.
have a look at this for inspiration

https://github.com/ITISFoundation/osparc-simcore/blob/fc054d6c76ee60d1827a74b31baa68191c4a1bc3/services/dynamic-scheduler/tests/unit/api_rpc/test_api_rpc__services.py#L572-L583

Reply:
Yes, that is another story. I tested the server side here.